.github/workflows/sonar-cloud.yml

```diff
@@ -0,0 +1,27 @@
+name: SonarCloud
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+    tags: [v*]
+  pull_request:
+
+jobs:
+  sonar_cloud:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Download artifacts
+        uses: actions/download-artifact@v4
+
+      - name: SonarCloud
+        if: always()
```

Comment:
Why the if? It doesn't seem like this does anything and this isn't the sonarcloud documentation.